### PR TITLE
Adds basic FFaker::Address.time_zone support

### PR DESCRIPTION
**Pull request by dugjason**

### CODE REVIEW

File: test/test_address.rb [hunk 1]
```diff
@@ -83,4 +83,8 @@ def test_country_code_of_particular_country
     assert_match 'UA', FFaker::Address::country_code('Ukraine')
     assert_match /[A-Z]{2}/, FFaker::Address::country_code('Foo')
   end
+
+  def test_time_zone
+    assert_match /[ a-z]+/, FFaker::Address::time_zone
```
Comment (marocchino) [inline test/test_address.rb]: This test is pointless.
How about `assert_include FFaker::Address::TIME_ZONE, FFaker::Address. time_zone` or `/\A[a-z_\/]\z/i`?

**Comment by dugjason** [comment]
Thanks for the review @marocchino. I've added both the ```assert_include FFaker::Address::TIME_ZONE``` comparison, and a more sensible regex.

**Comment by marocchino** [comment]
@dugjason Thanks, but we don't need both because It's too verbose. >_< Could you choose one of them please?

**Comment by dugjason** [comment]
Sure thing. I just included both examples for completeness, but I see it's one of the only tests with more than one test case.

**Comment by marocchino** [comment]
:+1:

**Pull request merged**